locations/spiders/denmans_gb.py

```diff
@@ -0,0 +1,53 @@
+from scrapy import Spider
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+
+
+class DenmansGBSpider(Spider):
+    name = "denmans_gb"
+    item_attributes = {"brand": "Denmans", "brand_wikidata": "Q116508855"}
+    # This seems to return all stores regardless of lat long as long as it's in the UK?
+    start_urls = ["https://www.denmans.co.uk/den/store-finder/findNearbyStores?latitude=51&longitude=-0"]
+
+    def parse(self, response):
+        for store in response.json()["results"]:
+            store["address"]["street_address"] = ", ".join(
+                filter(
+                    None,
+                    [
+                        store["address"].pop("line1"),
+                        store["address"].pop("line2"),
+                        store["address"].pop("line3"),
+                    ],
+                )
+            )
+            store["address"]["country"] = store["address"]["country"]["isocode"]
```

Comment:
Is it worth trying ["country"]["isocode"] more generally as this pattern seems to be present in a few scrapers?

Reply:
I agree with this. There might be a slight chance it breaks another spider, but that's why I prefer to clean up data before it goes into DictParser, not after. We can keep an eye on it with the next weekly.